jac-uk/admin
# Virus scanner status on releases page

## What's included?
The result of `runScannerTest` (see [digital-platform 1138](https://github.com/jac-uk/digital-platform/pull/1138)) is reflected on the releases page as 'Virus Scanner functionality: '.
This is to allow us to see the results of the scheduled test of the scan functionality.

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
Visit the releases page and observe the scan result. 
(for developers) amend the field in the database and ensure the result is reflected.

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
![image](https://github.com/user-attachments/assets/4f70dc07-a575-49ad-b52c-2f06f6815bb0)

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
